.github/dependabot.yml

```diff
@@ -59,3 +59,17 @@ updates:
       - "daniel-almeida-konkconsulting"
     labels:
       - "dependencies"
+
+  ##### Docker #####
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
are all dockerfiles on top-level?

Reply:
I hope that the solution I provided will fix it. It makes use of a feature that is currently in open beta (https://github.blog/changelog/2024-04-29-dependabot-multi-directory-configuration-public-beta-now-available/)